src/libs/utils.js

```diff
@@ -576,7 +576,20 @@ export const getSearchFilterFunctions = () => {
         includes(lowerCaseTerm, toLower(phrase))
       )([partialDarCode, ...(projectTitle.split(' ')), (updateDate || createDate)]);
       return !isNil(matched) && (draft !== false || draft !== 'false');
-    })(targetList)
+    })(targetList),
+    users: (term, targetList) => {
+      const lowerCaseTerm = toLower(term);
+      const isMatch = (userField) => includes(lowerCaseTerm, toLower(userField));
+
+      return filter(user => {
```

Comment:
You'll need to add the user's roles alongside displayName and email as part of the searchable terms (since they're visible on the table).